openvox/install.md

```diff
@@ -50,28 +53,62 @@ Then install the packages you want.
 If you have backed up config files, then restore them now.
 
 ### 🎉 That's it! You're done!
-Feel free to read the rest of the page for more options.
+Feel free to read the rest of the page for more options, especially if you're
+also intending to use Foreman.
 
 -----
 
 ## Advanced Options
 
-### Alternative Puppet uninstallation options
+### Installing Foreman with OpenVox
 
-There are some cases in which you might have to take more steps to safely remove legacy Puppet from your system.
+> 🚨 Tip: When using Foreman and OpenVox together, you must install OpenVox first.
+> See [Porting a Foreman infra to OpenVox](#porting-a-foreman-infra-to-openvox) for instructions on how to recover if you started with Foreman.
+{: class="alert alert-warning callout" }
 
-* If you're migrating from Puppet Enterprise you can use the `puppet-enterprise-uninstaller` script on each node as described in [their docs](https://www.puppet.com/docs/pe/latest/uninstalling.html).
-* If you're using distro provided packages, then you might have a bigger job.
-  For example, if you're using Debian packages, you may have several Puppet modules packaged as `.deb` packages that you'll have to move to your `Puppetfile`.
-  You might consider waiting until your distro packages OpenVox.
-    * If you do want to migrate now, then remove any puppet packages and dependencies you have installed using your distro tools.
-        * Debian family
-          * `apt autoremove <packagename>`
-        * RedHat family
-          * `yum autoremove <packagename>`
-    * You might also consider (carefully) cleaning up unused dependencies afterwards by running `apt` or `yum` autoremove without a package name.
+There are two ways to install Foreman: via the `foreman-installer` or via the `theforeman-foreman` module. ***In both cases, you must install OpenVox first***.
 
-You do not need to remove the `[apt|yum].puppet.com` repositories although the only thing you'll be able to use them for going forward is installing historical Puppet releases.
+#### Using the `foreman-installer`
+
+Edit `/etc/foreman-installer/custom-hiera.yaml` to include these Hiera settings.
+If you intend to continue managing your Foreman installation with puppet code, then follow the instructions in the [Managing OpenVox with OpenVox](#managing-openvox-with-openvox) sections to persist these settings into your standard Hiera data files.
+
+``` yaml
+---
+puppet::client_package: openvox-agent
+puppet::server_package: openvox-server
```

Comment:
These are answers and shouldn't be overridden via Hiera. The real way is to provide them options:

```
foreman-installer --puppet-client-package openvox-agent --puppet-server-package openvox-server
```

This also should solve your "caching" issue with the answers.

Reply:
Hrm. This gets complex when describing how to do something as simple as including the `puppetdb` plugin. As far as I can tell, there's no CLI options for 

``` yaml
puppetdb::puppetdb_package: openvoxdb
puppetdb::master::config::terminus_package: openvoxdb-termini
```